scripts/validate_docstrings.py

```diff
@@ -60,6 +60,7 @@
 PRIVATE_CLASSES = ["NDFrame", "IndexOpsMixin"]
 DIRECTIVES = ["versionadded", "versionchanged", "deprecated"]
 DIRECTIVE_PATTERN = re.compile(rf"^\s*\.\. ({'|'.join(DIRECTIVES)})(?!::)", re.I | re.M)
+BULLET_LIST_PATTERN = re.compile(r"(?<=:)(\*|-|\+|•|‣|⁃)\s.*")
```

Comment:
```suggestion
BULLET_LIST_PATTERN = re.compile(r"(\*|-|\+|•|‣|⁃)\s+.*")
```

I think should add `+` for the whitespace. Is there a reason for the positive lookbehind?